public/homepage/scripts/gallery.js

```diff
@@ -188,12 +188,15 @@ define(["jquery", "analytics"], function($, analytics) {
           newItem.find(".description").text(activity.description);
 
           // Check if activity_url ends with a slash, if it doesn't - add one before adding "remix"
-          var remix = "remix";
+          /*var remix = "remix";
           var endsWithSlash = (activity.url.charAt(activity.url.length-1) === "/");
           if(!endsWithSlash) {
             remix = "/remix";
           }
           newItem.find(".remix").attr("href", activity.url + remix);
+          */
+          //WIP: see the discussion on: https://github.com/mozilla/thimble.mozilla.org/issues/2012
+          newItem.find(".remix").attr("href", "https://thimble.mozilla.org/projects/" + (activity.url).match(/\d+/)[0] + "/remix");
```

Comment:
I'm going to suggest another fix here because regexing urls can be brittle.
@ZenanZha open up a PR against the https://github.com/mozilla/thimble-homepage-gallery repo and add a property to each object called `projectID` and let its value be the number at the end of the `url` property. FOr e.g., 
```js
{
  "title" : "Back to School Postcard",
  "thumbnail_url" : "https://thimble.mozilla.org/img/thumbnail-postcard.png",
  "url" : "https://thimbleprojects.org/mozillalearning/220079/",
  "description" : "Add your own content to this interactive postcard.",
  "tags" : ["images","code","javascript","interactive","js"],
  "teaching_kit_url" : "https://thimbleprojects.org/mozillalearning/62/",
  "featured" : true,
  "author" : "mozilla",
  "author_url" : "http://learning.mozilla.org"
}
```
would change to
```js
{
  "title" : "Back to School Postcard",
  "thumbnail_url" : "https://thimble.mozilla.org/img/thumbnail-postcard.png",
  "url" : "https://thimbleprojects.org/mozillalearning/220079/",
  "projectID" : 220079,
  "description" : "Add your own content to this interactive postcard.",
  "tags" : ["images","code","javascript","interactive","js"],
  "teaching_kit_url" : "https://thimbleprojects.org/mozillalearning/62/",
  "featured" : true,
  "author" : "mozilla",
  "author_url" : "http://learning.mozilla.org"
}
```

Then, out here, change this line to:
```js
newItem.find(".remix").attr("href", "/projects/" + activity.projectID + "/remix");
```